website/content/docs/install.mdx

```diff
@@ -17,6 +17,9 @@ There are several options to install Vault:
 
 1. [Helm for Kubernetes](https://www.vaultproject.io/docs/platform/k8s/helm)
 
+This is a test paragraph. Rather than working through install steps, you could try HCP Vault for free.
+
+<TryHcpCallout product="vault" />
```

Comment:
This is great! The only potential weirdness is the variance in our hcp capitalization callout names (HCPCallout in cloud docs). [Although these ](https://github.com/search?q=repo%3Ahashicorp%2Fcloud.hashicorp.com+HCPCallout&type=code)are only used on cloud docs I think so probably not a huge deal. I'm not sure if there is an author preference but I know it will be difficult to adjust later so wanting to bring it up.

Reply:
+1 would be curious on author preference. Added some thoughts in [this thread](https://github.com/hashicorp/dev-portal/pull/1377#discussion_r1029858318).